src/bin/pg_dump/pg_dump.c

```diff
@@ -12388,11 +12388,21 @@ dumpFunc(Archive *fout, const FuncInfo *finfo)
 	else
 		keyword = "FUNCTION";	/* works for window functions too */
 	
-	is_tsql_mstvf = isTsqlMstvf(fout, finfo, prokind[0], proretset[0] == 't');
+	pltsql_tvf_type = getTsqlTvfType(fout, finfo, prokind[0], proretset[0] == 't');
 
-	if (is_tsql_mstvf)
-		appendPQExpBufferStr(q,
+	switch (pltsql_tvf_type)
```

Comment:
can be move to babel files as a function with one bool to differentiate SET/RESET

Reply:
Yup, moved it to babel dump file